mesonbuild/interpreter.py

```diff
@@ -2454,8 +2460,10 @@ def do_subproject(self, dirname, kwargs):
     def get_option_internal(self, optname):
         # Some base options are not defined in some environments, return the
         # default value from compilers.base_options in that case.
-        for d in [self.coredata.base_options, compilers.base_options,
-                  self.coredata.builtins, self.coredata.compiler_options]:
+        for d in chain(
+                [self.coredata.base_options, compilers.base_options,
+                      self.coredata.builtins],
```

Comment:
__[Flake8]__ [E127] continuation line over-indented for visual indent

<sub>posted by Sider</sub>